spine-header-item.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2000-2018 TeamDev. All rights reserved.
+ * TeamDev PROPRIETARY and CONFIDENTIAL.
+ * Use is subject to license terms.
+ */
+
+import '@polymer/iron-flex-layout/iron-flex-layout.js';
+import { SpineAbstractItem } from './spine-abstract-item.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+
+/**
+ * `spine-header-item` can be used inside `paper-listbox` among its children to display a
```

Comment:
Is this really true with Polymer 3 `paper-listbox`? Just a point to discuss later.

Reply:
Polymer 3.0 just changes the "packaging" aspects (npm & ES6 modules), but retains element APIs as is, so the scenarios of how this element is used remain the same.